iml-wire-types/src/high_availability.rs

```diff
@@ -29,6 +33,19 @@ impl TryFrom<&str> for NodeType {
     }
 }
 
+impl Display for NodeType {
```

Comment:
Just something to think about since we've simplified our errors with `this_error`. There is something similar for deriving display for enums: https://jeltef.github.io/derive_more/derive_more/